test/smart-wallet/gasEstimations.ts

```diff
@@ -470,7 +472,7 @@ describe("Wallet tx gas estimations with and without refunds", function () {
       to: token.address,
       // value: ethers.utils.parseEther("1"),
       data: encodeTransfer(charlie, ethers.utils.parseEther("10").toString()),
-      nonce: await userSCW.getNonce(0),
+      nonce: await userSCW.getNonce(EOA_CONTROLLED_FLOW),
```

Comment:
I see now. should we keep this getNonce method?

Reply:
ok we will have to use getNonce(1) moving forward for 'forward' transaction payload

Reply:
default nonce() method would return nonces[0] and getNonce(batchId) would return nonces[bacthId]

Reply:
yep